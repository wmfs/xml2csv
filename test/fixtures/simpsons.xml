<People>
    <Person>
        <Name>Maggie</Name>
        <Age>3</Age>
        <Gender>Female</Gender>
        <Siblings>
            <Brother>Bart</Brother>
            <Sister>Lisa</Sister>
        </Siblings>
    </Person>
    <Person>
        <Name>Lisa</Name>
        <Age>3</Age>
        <Gender>Female</Gender>
        <Siblings>
            <Brother>Bart</Brother>
            <Sister>Maggie</Sister>
        </Siblings>
    </Person>
    <Person>
        <Name>Marge</Name>
        <Age>45</Age>
        <Gender>Female</Gender>
    </Person>
    <Person>
        <Name><![CDATA[Homer]]></Name>
        <Age>47</Age>
        <Gender>Male</Gender>
    </Person>
</People>
