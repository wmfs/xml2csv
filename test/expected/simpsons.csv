name,age,gender,brother,sister
"Maggie",3,"Female","Bart","Lisa"
"Lisa",3,"Female","Bart","Maggie"
"Marge",45,"Female",,
"Homer",47,"Male",,
